### ISSUE TITLE
How to get suggestions in v2?

### ISSUE THREAD

**Issue by pablobarrera** [VictorRancesCode/flutter_dialogflow#15]
Hi, I have some doubts about how I should show suggested replies to the user.

I noticed in Dialogflow that some specific platforms like Google, Facebook, Slack, Skype have suggestion chips or quick replies, also I see that it's possible to add a custom payload with custom JSON. So, if I'm making a chatbot in my Flutter app, here are my questions:

- Should I use the responses from these platforms to get suggestions?
- Should I set custom suggestions in payload and get them from it?
- If I want to get suggestions from each platform or from payload, how can I access to them using dialogflow_v2?

**Comment by TheBoy-WhoCode** [comment]
@pablobarrera did you got the answer of this issue?

**Comment by thejitenpatel** [comment]
@pablobarrera did you get a solution to these?

**Comment by pablobarrera** [comment]
I found a way, you can see it here:
https://stackoverflow.com/questions/58341273/how-to-create-a-chatbot-with-response-options-in-flutter/58679095#answer-58679095

> In the Dialogflow console you could add a custom payload to your messages, like this:
> 
> `{"suggestions": ["Reply 1", "Reply 2"]}`

**Comment by thejitenpatel** [comment]
@pablobarrera yeah actually I tried this code but shows me in the console and I'm not able to know should I embed that in widgets. So is there any way to do that as they have shown on StackOverflow?
